src/component/child/index.js

```diff
@@ -31,6 +31,10 @@ export class ChildComponent extends BaseComponent {
         this.onProps = this.tryCatch(options.onProps || noop, false);
         this.onError = this.tryCatch(options.onError || (err => { throw err; }));
 
+        // watchforResize looks out for change in size of the contents inside of an iframe.
+        // It autoresizes the window if required.
+        this.watchForResize();
```

Comment:
Can we do this in the `init()` method instead? Trying to keep instantiating and initializing the component to two different steps.